src/app/api/stripe/sync.ts

```
@@ -24,9 +28,57 @@ export async function sync_stripe_data(
+
+    const subscription = subscriptions.data[0]
+    if (!subscription) {
+        const subData = { status: 'none' } satisfies StripeSubData
+        await redis.set(get_redis_key('stripe:customer', customer_id), subData)
+        return subData
+    }
+
+    const subData = {
+        subscription_id: subscription.id,
+        status: subscription.status,
+        price_id: subscription.items.data[0]!.price.id,
```

Biome (1.9.4)

[error] Forbidden non-null assertion.

Unsafe fix: Replace with optional chain operator ?. This operator includes runtime checks, so it is safer than the compile-only non-null assertion operator


(lint/style/noNonNullAssertion)